icu4c/source/i18n/unitconverter.h

```diff
@@ -7,15 +7,71 @@
 #ifndef __UNITCONVERTER_H__
 #define __UNITCONVERTER_H__
 
-
-#include <vector>
+#include "unicode/errorcode.h"
+#include "unicode/measunit.h"
+#include "unicode/measure.h"
+#include "unicode/stringpiece.h"
 
 U_NAMESPACE_BEGIN
+// Data Skeleton.
 
+enum Constants {
```

Comment:
Does this have to be in the header file? Could it be moved to unitconverter.cpp?
- Since I think we should move to data-driven constants, I'm not sure we can ever have an exhaustive list here.
- I don't think users of this library will use these enums/constants directly?

Reply:
If this enum is intended for use outside of unitconverter.cpp, then it should be in a .h file.  Otherwise, it can be moved to unitconverter.cpp.

Note that this .h file is internal; this isn't exposed to public users.

Reply:
Good point ... I put it in the .h file because it is related to external data. However, I will move it to the .cpp file.